target/linux/ramips/image/mt7621.mk

```diff
@@ -341,6 +365,27 @@ define Device/beeline_smartbox-turbo
 endef
 TARGET_DEVICES += beeline_smartbox-turbo
 
+define Device/belkin_rt1800
+  $(Device/dsa-migration)
```

Comment:
As this device has not been previously available prior to the switch to DSA there is not need for this line imho.

Reply:
I have no idea what dsa-migration is. The WAX202 (support merged a few weeks ago) has it.

Reply:
The `$(Device/dsa-migration)` symbol is used to warn users of **existing installations** about configuration incompatibility when upgrading from non-DSA OpenWrt to DSA OpenWrt. It should hence only be added to devices which were already in our tree before the ramips/mt7621 target has been migrated to DSA.

Reply:
Removed.

Reply:
We still need it, due to `compat_version` being set to `1.1` by default, so upgrading via LuCI will show a warning if we don't call `$(Device/dsa-migration)`.
https://github.com/openwrt/openwrt/blob/b8f8c6f2dd8d47216117cb5b78184531ab21dddd/target/linux/ramips/mt7621/base-files/etc/board.d/05_compat-version#L10-L14

Reply:
Readded